docs/docs/References/Glossary.md

```diff
@@ -28,6 +28,13 @@ The process by which the Delphix Engine learns about how a particular environmen
 ## dSource
 See [Linked Dataset](#linked-dataset)
 
+## Empty VDB
+A VDB that is created from scratch, without provisioning from another dataset. Users can create empty VDBs when they want to construct a brand-new dataset from within Delphix, instead of creating it externally and then ingesting it.
+
+This "empty" VDB, of course, will typically not stay empty for long. Data will be added as users work with the new dataset.
```

Comment:
kinda nit, I'm not sure I like this sentence but my suggestions aren't much better. Maybe instead of saying like ths vdb will typically not stay empty for long we can write a more clear sentence about how it starts out empty but can have things written into it depending on the user?

You can leave it this way if too if you disagree with my suggestion.